src/Forms/__tests__/Dropdown.test.tsx

```diff
@@ -415,7 +415,7 @@ describe('Dropdown', function () {
     });
     it('the label value is hidden in the UI', function () {
       const style = window.getComputedStyle(getByText('semesters'));
-      strictEqual(style.display, 'none');
+      strictEqual(style.overflow, 'hidden');
```

Comment:
Here and in the `TextInput` test, I think we want to be testing that we can get a reference to the input with `getByLabelText('InvisibleLabel')`, since that will provide better confirmation that the accessibility features are working -- I'm less concerned about testing individual css rules (and, as noted elsewhere, `window.getComputedStyle` doesn't always work correctly).